packages/expression-parser/src/expression-parser/customFunctions.js

```diff
@@ -32,6 +33,8 @@ const translate = (value, translations) => {
 
 const date = (...argumentList) => new Date(...argumentList);
 
+const dateToString = date => fns.format(date, 'yyyy-MM-dd');
```

Comment:
There's probably a greater conversion about standardising date strings in tupaia to be had. But for now adding this utility function I think will help